src/modules/facility-loan/dto/create-facility-loan-request.dto.ts

```diff
@@ -106,8 +106,8 @@ export class CreateFacilityLoanRequestItem {
   readonly yearBasis: YearBasisCodeEnum;
 
   @ValidatedStringApiProperty({
-    description: 'The frequency which the rate will change.',
-    required: false,
+    description: `The frequency which the rate will change. This is required if productTypeGroup is ${ProductTypeGroupEnum.EWCS}.`,
```

Comment:
Could we change this to `The frequency _with_ which the rate will change.`

Reply:
Done!